codebuild/cd/manylinux-x64-build.yml

```diff
@@ -11,16 +11,21 @@ phases:
     commands:
       - cd aws-crt-java
       - git submodule update --init
-      - JAVA_HOME=/opt/java-se-8u40-ri/ mvn -B compile
+      # use a fix deploy version for platform specific jar
+      - mvn versions:set -DnewVersion=deploy
+      - JAVA_HOME=/opt/java-se-8u40-ri/ mvn -B install -DskipTests -Plinux-x64
+      - ls target
   post_build:
     commands:
       # get the shared libs from the native build
       - mkdir -p ../dist
       - cp -rv target/cmake-build/lib ../dist/
-
+      # get the platform specific jar with classifier
+      - cp target/*.jar ../dist/
```

Comment:
The image used doesn't have aws-cli installed. Still use codebuild to transfer the artifacts.